app/js/lib/store_update_handling.js

```diff
@@ -48,7 +48,7 @@ module.exports = {
 			viewHandlers.forEach( function ( viewHandlerConfig ) {
 				viewHandlerConfig.viewHandler.update.call(
 					viewHandlerConfig.viewHandler,
-					state[ viewHandlerConfig.stateKey ]
+					_.get( state, viewHandlerConfig.stateKey )
```

Comment:
To allow for accessing values that are nested somewhere in the global object. See DonationForm initialization for examples, where the element slider view handlers needs a simple value instead of the whole `formValues` object.